README.md

```diff
@@ -85,7 +91,7 @@ npm install
 
 ## Samples
 
-[Samples README](samples)
+[Samples README](samples/README.md)
```

Comment:
Minor: Not related to this line, but can you add/adjust the PubSub sample to support MQTT5? The other 3 SDKs have MQTT5 PubSub support - so for consistency it would be nice to have it in Javascript too.

Reply:
There's a PubSub sample for MQTT5 in the CRT.  I can add links in appropriate spots, but I don't have any desire to duplicate it.